### PR TITLE
[AT] - update readme with ES Modules instructions

### PR DESCRIPTION
**Pull request by andrew-c-tran**
Updated Readme.md to contain instructions on how to use package with ES module importer. Addresses issue #153 

Verified in Babel 7.4

### CODE REVIEW

**Comment by andrew-c-tran** [comment]
Thanks for approving @IDisposable . I don't have write access to this repo so you'll have to merge it :)

File: README.md [hunk 1]
```diff
@@ -13,6 +13,13 @@ Basic usage
 
 This package is available as *humanize-duration* on [npm](https://www.npmjs.com/package/humanize-duration) and Bower. You can also include the JavaScript file in the browser.
 
+With `ES Modules`:
```
Comment (EvanHahn) [inline README.md]: Why is this in quotes and not simply "ECMAScript modules" as text?

File: README.md [hunk 2]
```diff
@@ -13,6 +13,13 @@ Basic usage
...
+
+```js
+import * as humanizeDuration from 'humanize-duration';
```
Comment (EvanHahn) [inline README.md]: `humanize-duration` doesn't actually use ES exports...does this kind of thing work on _all_ build setups?

**Comment by EvanHahn** [comment]
@andrew-c-tran Bumping this. Have you had a chance to look at my comments?

**Comment by EvanHahn** [comment]
Closing because I haven’t heard back in months. Let me know if that’s wrong and I’m happy to reopen!

**Pull request closed without merging**